.github/workflows/automatic-doc-checks.yml

```diff
@@ -1,9 +1,12 @@
 name: Main Documentation Checks
 
 on:
-  - push
-  - pull_request
-  - workflow_dispatch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
Can we leave the pull request trigger unrestricted?
We currently have another working branch apart from main to which we open PRs, and they should be checked.